stacks-signer/src/tests/conf/signer-2.toml

```diff
@@ -1,5 +0,0 @@
-
-stacks_private_key = "b169d0d1408f66d16beb321857f525f9014dfc289f1aeedbcf96e78afeb8eb4001"
-node_host = "127.0.0.1:20443"
-endpoint = "localhost:30002"
-network = "testnet"
```

Comment:
I didn't see this and `signer-3.toml` being used anywhere so I purged them. @jferrant let me know if these are needed for anything.

Reply:
I don't mind you purging them for sure! Thehy were originally there as demonstrations for the testnet guys, but one should be sufficient/I think its the only one we use throughout the test suites/stacks node tests.

Reply:
Nice, thanks :ok_hand: 

> but one should be sufficient

Note: with this change we use two different files (`signer-0.toml` and `signer-1.toml`) in the tests that conflicted, just so that they don't try to spin up two MockServerClient with the same `node_host`.